src/dmd/link.d

```diff
@@ -960,21 +960,24 @@ version (Windows)
         const(char)* linkOptions(bool x64)
         {
             OutBuffer cmdbuf;
-            if (auto p = getVCDir(VCDir.Lib, x64))
+            if (auto vclibdir = getVCDir(VCDir.Lib, x64))
+            if (vclibdir)
```

Comment:
Isn't this `if` unnecessary?

Reply:
Ooops, merge error. Removed and amended.